tests/rptest/services/redpanda_installer.py

```diff
@@ -386,21 +391,37 @@ def released_versions(self):
             if len(self._released_versions) > 0:
                 return self._released_versions
 
-            releases_json = self._released_versions_json()
             versions = []
-            for release in releases_json:
-                match = VERSION_RE.findall(release["tag_name"])
-                if match:
-                    versions.append(int_tuple(match[0]))
-                else:
-                    if release["tag_name"].startswith("release-"):
-                        # Tags like 'release-20.12.4' predate the modern Redpanda versioning scheme
-                        self._redpanda.logger.info(
-                            f"Ignoring legacy release {release['tag_name']}")
+
+            if releases_env := os.getenv(RP_GIT_RELEASED_VERSIONS):
+                # releases provided with an environment variable, parse and save it
+                self._redpanda.logger.debug(
+                    f"getting released_versions from environment variable {RP_GIT_RELEASED_VERSIONS}={releases_env}"
+                )
+                for release in releases_env.split(":"):
+                    if match := STRICT_VERSION_RE.findall(release):
+                        versions.append(int_tuple(match[0]))
                     else:
                         self._redpanda.logger.warn(
-                            f"Malformed release tag in repo: {release['tag_name']}"
+                            f"Malformed release tag in {RP_GIT_RELEASED_VERSIONS}: '{release}'"
                         )
```

Comment:
Is there a corresponding vtools change to go along with this? Will it already handle pagination? Or are the releases not coming directly from Github?

Reply:
Ah I see https://github.com/redpanda-data/vtools/issues/2171